### ISSUE TITLE
Problems with Scala 2.12.5, anyone?

### ISSUE THREAD

**Issue by akauppi** [lloydmeta/enumeratum#183]
I'm getting an enumeratum (or circe-enumeratum) related error, upgrading a project to 2.12.5. 2.12.4 works fine.

Are others seeing this too? Is anyone able to use enumeratum with Scala 2.12.5?

Unfortunately don't have the time to debug further, right now.

```
$ sbt test
[info] Loading settings from idea.sbt,plugins.sbt ...
[info] Loading global plugins from /Users/asko/.sbt/1.0/plugins
[info] Loading settings from plugins.sbt ...
[info] Loading project definition from /Users/asko/Git/nm.akauppi/project
[info] Loading settings from build.sbt ...
[info] Set current project to nm (in build file:/Users/asko/Git/nm.akauppi/)
[info] Compiling 2 Scala sources to /Users/asko/Git/nm.akauppi/data/target/scala-2.12/classes ...
[error] /Users/asko/Git/nm.akauppi/data/src/main/scala/Meeting.scala:114:18: macro implementation not found: findValues
[error] (the most common reason for that is that you cannot use macro implementations in the same compilation run that defines them)
[error]     val values = findValues
[error]                  ^
[error] one error found
[error] (data / Compile / compileIncremental) Compilation failed
[error] Total time: 9 s, completed 21 Mar 2018, 11.38.09
```

enumeratum 1.5.13
enumeratum-circe 1.5.16

**Comment by lloydmeta** [comment]
Are you by any chance using Java9? If so, it might be related to this issue https://github.com/scala/scala-dev/issues/480

**Comment by akauppi** [comment]
Spot on. :) Thanks for the pointer. 

Do you want to keep this issue open (to gather those coming with the same problem, and avoid new issues being created), or close the issue?

**Comment by lhohan** [comment]
This issue can be closed, using Scala 2.12.6 fixes the it.
